infra/modules/worklytics-psoxy-connection-generic/main.tf

```diff
@@ -0,0 +1,35 @@
+
+locals {
+  # for backwards compatibility < 0.4.6
+  instance_id = coalesce(var.psoxy_instance_id, var.display_name)
+
+  # build TODO
+  last_step                     = 5 + (length(var.settings_to_provide) > 0 ? 1 : 0) + 1
+  per_setting_instructions      = [for k, v in var.settings_to_provide : "    - Copy and paste `${v}` as the value for \"${k}\"."]
+  per_setting_instructions_text = length(var.settings_to_provide) > 0 ? "\n${join("\n", tolist(local.per_setting_instructions))}" : ""
+}
+
+resource "local_file" "todo-worklytics-connection" {
```

Comment:
This is already present in `worklytics-psoxy-connection` module; not sure if intended or it is duplicated

Reply:
yeah, it's a more general case. the other one has a `url` param that's not relevant for the bulk case.